<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1778</width>
    <height>981</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>840</y>
      <width>1711</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1,0,1,0,0">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>图片名</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="picturename"/>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>图片大小</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="picturesize"/>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>放大率</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="magnification"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1711</width>
      <height>821</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="3,1">
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="tabPosition">
        <enum>QTabWidget::South</enum>
       </property>
       <property name="tabShape">
        <enum>QTabWidget::Rounded</enum>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tab">
        <attribute name="title">
         <string>原始图片</string>
        </attribute>
        <widget class="MLabel" name="label_3">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>1</y>
           <width>1271</width>
           <height>791</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="tab_2">
        <attribute name="title">
         <string>人工标签</string>
        </attribute>
        <widget class="MLabel" name="label_4">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>1</y>
           <width>1271</width>
           <height>791</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="tab_3">
        <attribute name="title">
         <string>预测标签</string>
        </attribute>
        <widget class="MLabel" name="label_6">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>1</y>
           <width>1271</width>
           <height>791</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </widget>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>  当前图片统计</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableWidget" name="tableWidget">
         <row>
          <property name="text">
           <string/>
          </property>
         </row>
         <row>
          <property name="text">
           <string>1</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>2</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>3</string>
          </property>
         </row>
         <column>
          <property name="text">
           <string>1</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>2</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>3</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>4</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>5</string>
          </property>
         </column>
         <item row="0" column="0">
          <property name="text">
           <string>类别序号</string>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
         </item>
         <item row="0" column="1">
          <property name="text">
           <string>类别名称</string>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
         </item>
         <item row="0" column="2">
          <property name="text">
           <string>像素数/%</string>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
         </item>
         <item row="0" column="3">
          <property name="text">
           <string>正检/%</string>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
         </item>
         <item row="0" column="4">
          <property name="text">
           <string>错检/%</string>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
         </item>
         <item row="1" column="0">
          <property name="text">
           <string/>
          </property>
         </item>
         <item row="3" column="0">
          <property name="text">
           <string>总体统计</string>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>  所有图片统计</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableWidget" name="tableWidget_2">
         <row>
          <property name="text">
           <string/>
          </property>
         </row>
         <row>
          <property name="text">
           <string>1</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>2</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>3</string>
          </property>
         </row>
         <column>
          <property name="text">
           <string>1</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>2</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>3</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>4</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>5</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>6</string>
          </property>
         </column>
         <item row="0" column="0">
          <property name="text">
           <string>类别序号</string>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
         </item>
         <item row="0" column="1">
          <property name="text">
           <string>类别名称</string>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
         </item>
         <item row="0" column="2">
          <property name="text">
           <string>像素数/%</string>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
         </item>
         <item row="0" column="3">
          <property name="text">
           <string>正检/%</string>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
         </item>
         <item row="0" column="4">
          <property name="text">
           <string>错检/%</string>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
         </item>
         <item row="0" column="5">
          <property name="text">
           <string>漏检/%</string>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1778</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menusaddfdgf">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="action"/>
    <addaction name="action_2"/>
    <addaction name="action_3"/>
   </widget>
   <addaction name="menusaddfdgf"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_4"/>
   <addaction name="action_5"/>
   <addaction name="action_6"/>
   <addaction name="action_7"/>
   <addaction name="action_8"/>
   <addaction name="action_9"/>
   <addaction name="action_10"/>
  </widget>
  <action name="action">
   <property name="text">
    <string>导入图片</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>导入人工标签</string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>导入预测标签</string>
   </property>
  </action>
  <action name="action_4">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/icon/images/leftarrows.jpg</normaloff>:/icon/images/leftarrows.jpg</iconset>
   </property>
   <property name="text">
    <string>上一张</string>
   </property>
  </action>
  <action name="action_5">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/icon/images/rightarrows.jpg</normaloff>:/icon/images/rightarrows.jpg</iconset>
   </property>
   <property name="text">
    <string>下一张</string>
   </property>
  </action>
  <action name="action_6">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/icon/images/magnify.jpg</normaloff>:/icon/images/magnify.jpg</iconset>
   </property>
   <property name="text">
    <string>放大</string>
   </property>
  </action>
  <action name="action_7">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/icon/images/reduce.jpg</normaloff>:/icon/images/reduce.jpg</iconset>
   </property>
   <property name="text">
    <string>缩小</string>
   </property>
  </action>
  <action name="action_8">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/icon/images/wholeshow.png</normaloff>:/icon/images/wholeshow.png</iconset>
   </property>
   <property name="text">
    <string>全局显示</string>
   </property>
  </action>
  <action name="action_9">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/icon/images/refresh.png</normaloff>:/icon/images/refresh.png</iconset>
   </property>
   <property name="text">
    <string>刷新显示表格</string>
   </property>
  </action>
  <action name="action_10">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/icon/images/category.jpg</normaloff>:/icon/images/category.jpg</iconset>
   </property>
   <property name="text">
    <string>自动生成类别</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MLabel</class>
   <extends>QLabel</extends>
   <header location="global">Mlabel.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
